File: epargne/DTO/CompteEpargneDto.cs
using System;
using System.Collections.Generic;

namespace Epargne.DTO
{
    public class CompteEpargneDto
    {
        public int IdCompte { get; set; }
        public int IdClient { get; set; }

        public decimal CapitalEpargne { get; set; }
        public string Libelle { get; set; }
        public DateOnly DateOuverture { get; set; }
        public decimal Retrait  { get; set; } = 50.00m;

        public decimal TauxInteret { get; set; }

        public List<TransactionEpargneDto> Transactions { get; set; }
    }
}

File: server-api/src/main/java/com/example/server_dtos/CompteEpargneDto.java
package com.example.server_dtos;

import java.io.Serializable;
import java.time.LocalDate;

import lombok.Data;

@Data
public class CompteEpargneDto implements Serializable {

    private Integer idClient;
    private Double capitalEpargne;
    private String libelle;
    private LocalDate dateOuverture = LocalDate.now();
    private Double retrait = 50.0;
    private Double tauxInteret = 2.1;

}
